frontend/src/components/BorderBox/BorderBox.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box } from '@highlight-run/ui'
+import React from 'react'
+
+const BorderBox: React.FC<React.PropsWithChildren> = ({ children }) => {
```

Comment:
Nice!

I could see this going into `@highlight-run/ui`.
cc @ccschmitz - related to our convos before about how we use the box component with so many props defined.

Reply:
Yeah thought about that while building these components. The components are very barebones Box with a few props but there is value in making them reusable since we use them really often